### PR TITLE
Add HoF entries

### PR DESCRIPTION
**Pull request by davidegirardi**

### CODE REVIEW

File: content/security-hall-of-fame/findings.toml [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+project = "Synapse"
+
+[[findings]]
+date = "2023-04-25"
+reporter.name = "S1m"
+reporter.link = "https://github.com/p1gp1g/"
+summary = """
+Discovered an XSS vector for
+[CVE-2023-30609](https://nvd.nist.gov/vuln/detail/CVE-2023-30609)/
+[GHSA-xv83-x443-7rmw[(https://github.com/matrix-org/matrix-react-sdk/security/advisories/GHSA-xv83-x443-7rmw).
```
Comment (MTRNord) [inline content/security-hall-of-fame/findings.toml:30]:
```suggestion
[GHSA-xv83-x443-7rmw](https://github.com/matrix-org/matrix-react-sdk/security/advisories/GHSA-xv83-x443-7rmw).
```

Comment (MTRNord) [reply content/security-hall-of-fame/findings.toml:30]: Diff is horrible: The change is just the flipped bracket

File: content/security-hall-of-fame/findings.toml [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+project = "Matrix React SDK"
+
+[[findings]]
+date = "2023-04-10"
+reporter.name = "Cadence Ember"
+reporter.link = "https://cadence.moe/"
+summary = """
+Found an HTML injection via highlighting of search results
+([CVE-2023-30609](https://nvd.nist.gov/vuln/detail/CVE-2023-38690)/
+[GHSA-xv83-x443-7rmw[(https://github.com/matrix-org/matrix-react-sdk/security/advisories/GHSA-xv83-x443-7rmw)).
```
Comment (MTRNord) [inline content/security-hall-of-fame/findings.toml:41]:
```suggestion
[GHSA-xv83-x443-7rmw](https://github.com/matrix-org/matrix-react-sdk/security/advisories/GHSA-xv83-x443-7rmw)).
```

Same as above. flipped bracket

**Review by MTRNord — CHANGES_REQUESTED** [review]
Some typos on the links

File: content/security-hall-of-fame/findings.toml [hunk 3]
```diff
@@ -1,3 +1,58 @@
+[[findings]]
+date = "2023-"
```
Comment (dkasak) [inline content/security-hall-of-fame/findings.toml:2]:
```suggestion
date = "2023-05-27"
```

File: content/security-hall-of-fame/findings.toml [hunk 4]
```diff
@@ -1,3 +1,58 @@
...
+date = "2023-02-18"
+reporter.name = "Val Lorentz"
+reporter.link = "https://valentin-lorentz.fr/"
+summary = """
+Discovered a IRC command injection via admin commands
+([CVE-2023-38690](https://nvd.nist.gov/vuln/detail/CVE-2023-38690)/
+[GHSA-3pmj-jqqp-2mj3](https://github.com/matrix-org/matrix-appservice-irc/security/advisories/GHSA-3pmj-jqqp-2mj3)).
+"""
+project = "matrix-appservice-irc"
+
```
Comment (dkasak) [inline content/security-hall-of-fame/findings.toml:55]: We usually sorted these chronologically, even when adding then in afterwards.

**Review by dkasak — CHANGES_REQUESTED** [review]

File: content/security-hall-of-fame/findings.toml (hunk 4 above)
Comment (davidegirardi) [reply content/security-hall-of-fame/findings.toml:73]: Fixed in dd0bcf88a3a7f54a7aef40586ded611302fa47bf

**Pull request merged**